draft-ietf-core-cf-reg-update.md

```diff
@@ -264,17 +206,93 @@ The preferred string format is as defined in {{Section 8.3.1 of -http-sema}} and
    Otherwise, a parameter value is included unquoted.
 1. A single semicolon character without any adjacent whitespace characters is used as the separator between Media Type and parameters.
 
-# Security Considerations
+### Examples for Invalid Registration Requests
 
-This document hardens the registration procedures of CoAP Content-Formats in ways that reduce the chances of malicious manipulation of the associated registry.
+This section provides examples of registration requests for the "CoAP Content-Formats" Registry that are invalid but would be approved under the procedure defined in {{Section 12.3 of -coap}}.
+The checklist defined in {{checks}} should prevent any of these attempts from succeeding.
+These examples serve as a representative, but not exhaustive, sample to train the DE's eye on invalid registration attempts.
 
-Other than that, it does not change the Security Considerations of {{-coap}}.
+All the example registration requests use two CoAP Content-Format identifiers: 64998 and 64999.
 
-# IANA Considerations
+For each of the following example registration requests, one can create a similar instance where the requested registration is for a CoAP Content-Format identifier within the "IETF Review or IESG Approval" range.
+Likewise, such registrations must not be allowed to succeed.
+
+#### The Media Type is Unknown {#ex-unknown-mt}
+
+The registrant requests an FCFS Content-Format ID for an unknown Media Type:
+
+| Content Type | Content Coding | ID |
+|--|--|--|
+| application/unknown+cbor | - | 64999 |
+{: align="left" title="Attempt at Registering Content-Format for an Unknown Media Type"}
+
+#### The Media Type Parameter is Unknown
+
+The registrant requests an FCFS Content-Format ID for an existing Media Type with an unknown parameter:
+
+| Content Type | Content Coding | ID |
+|--|--|--|
+| application/cose;unknown-parameter=1 | - | 64999 |
+{: align="left" title="Attempt at Registering Content-Format for Media Type with Unknown Parameter"}
+
+#### The Media Type Parameter Value is Invalid
+
+The registrant requests an FCFS Content-Format ID for an existing Media Type with an invalid parameter value:
+
+| Content Type | Content Coding | ID |
+|--|--|--|
+| application/cose;cose-type=invalid | - | 64999 |
+{: align="left" title="Attempt at Registering Content-Format for Media Type with Invalid Parameter Value"}
+
+#### The Content Coding is Unknown
+
+The registrant requests an FCFS Content-Format ID for an existing Media Type with an unknown Content Coding:
+
+| Content Type | Content Coding | ID |
+|--|--|--|
+| application/senml+cbor | inflate | 64999 |
```

Comment:
I still like the "inflate" encoding - one to propose next year on April 1st ? How could we inflate data in the most effective & efficient way... maybe the size should grow over time with Moore's law to fill the available network capacity.

Reply:
🤣